go.mod

```diff
@@ -1,9 +1,13 @@
 module github.com/terraform-providers/terraform-provider-github
 
 require (
-	github.com/google/go-github/v28 v28.1.1
+	github.com/google/go-github/v28 v28.1.1 // indirect
```

Comment:
Not sure if `v28` should still be there after the switch to `v29`.

Reply:
I haven't used `go mod vendor` before, but I can confirm that the string `v28` doesn't appear anywhere in this repository. Perhaps there is a transitive dependency on the old version?

Reply:
It may be true, you can check if `go mod tidy` will make a difference.

Reply:
It may be true, you can check if go mod tidy will make a difference.